### ISSUE TITLE
Using shift+Super+} for next_tab key binding doesn't work

### ISSUE THREAD

**Issue by AlanFoster** [gnome-terminator/terminator#326]
I'm trying to use `Shift+Super+{` and `Shift+Super+}` as keybindings for moving next_tab/previous_tab. I'm trying to use these keybindings as it mimics iterm's bindings.

![image](https://user-images.githubusercontent.com/1271782/102819190-f05ccb00-43ca-11eb-88f8-da7aed127757.png)

Unfortunately when I try to use the bindings, terminator only outputs the `{` and `}` characters instead of moving between tabs:

![image](https://user-images.githubusercontent.com/1271782/102819354-39148400-43cb-11eb-9db7-e264cb83c532.png)

I'm not sure if this is a bug, intentional functionality, or PEBKAC. I followed the guide [here](https://terminator-gtk3.readthedocs.io/en/latest/preferences.html#keybindings).

I've verified this behavior with:
- terminator 1.92
- Kali 2020.2
- Ubuntu 20.04

**Comment by mattrose** [comment]
Not a PBKAC.  this is #149 and it's fixed in 2.0. There's a PPA listed in INSTALL.md. Sorry about that

**Comment by AlanFoster** [comment]
Thanks @mattrose ! I confirmed the latest release does work, here's the steps I ran through incase it's useful :+1:

---

I just tried adding the ppa on Kali 2020.2 using the installation instructions:
```
sudo add-apt-repository ppa:mattrose/terminator
sudo apt-get update
sudo apt install terminator
```

After an `apt update` I got:
```
Err:6 http://ppa.launchpad.net/mattrose/terminator/ubuntu hirsute Release                      
  404  Not Found [IP: 91.189.95.85 80]
```

I then tried building from source following along with the install steps and got a `gettext` error:
```
$ python3 setup.py build
...
creating build/mo/ca@valencia
compiling po/ca@valencia.po -> build/mo/ca@valencia/terminator.mo
Building gettext files failed. Ensure you have gettext installed. Alternatively, try setup.py --without-gettext [build|install]
Error: [Errno 2] No such file or directory: 'msgfmt'
```

After running both the build and install with the suggested `--without-gettext` flag the install then fails:
```
running install_data
creating /usr/local/share/applications
error: can't copy 'data/terminator.desktop': doesn't exist or not a regular file
```

The data directory has similar files, but no `terminator.desktop` file:
```
$ ls -la data
total 24
drwxr-xr-x  4 kali kali 4096 Dec 21 19:07 .
drwxr-xr-x 11 kali kali 4096 Dec 21 19:07 ..
drwxr-xr-x  4 kali kali 4096 Dec 21 19:07 icons
drwxr-xr-x  2 kali kali 4096 Dec 21 19:07 layout-files-examples
-rw-r--r--  1 kali kali 2187 Dec 21 19:07 terminator.appdata.xml.in
-rw-r--r--  1 kali kali  433 Dec 21 19:07 terminator.desktop.in
```

In the end I manually changed the sources list:
```diff
- deb http://ppa.launchpad.net/mattrose/terminator/ubuntu hirsute main
+ deb http://ppa.launchpad.net/mattrose/terminator/ubuntu focal main
```

Then I had to install it with:
```
apt install terminator/focal
```

It didn't work immediately - I had to rebind the keys again.

Before:
![image](https://user-images.githubusercontent.com/1271782/102834893-23b05180-43ed-11eb-8679-32096cb087c4.png)

After:
![image](https://user-images.githubusercontent.com/1271782/102834912-30cd4080-43ed-11eb-8ca2-b3cb5d8dab40.png)

Then it worked, thanks! 🎉
